### PR TITLE
Delete all issue comments for a status check which was fixed.

### PR DESCRIPTION
**Pull request by rhuss**
This is WIP since I have no idea yet how to test this ;)

### CODE REVIEW

**Comment by rhuss** [comment]
@jimmidyson  how would one test this end2end ?

**Comment by jimmidyson** [comment]
Set up a test repo, add pure-bot integration to it and send a PR? It's a bit rubbish I know but it's all I can think of right now.

**Comment by rhuss** [comment]
Thanks. Can I use project ipaas-bots on dedicated to install it (in addition) ?

will try this evening ...

**Comment by jimmidyson** [comment]
Of course!

File: Dockerfile.in [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 FROM ARG_FROM
 
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+MAINTAINER Roland Huss <rhuss@redhat.com>
```
Comment (zregvart) [inline Dockerfile.in]: Cheeky, perhaps dev@syndesis.io

Comment (rhuss) [reply Dockerfile.in]: Damned, you got me while spreading breadcrumbs.

Changed this in the other PR, not sure whether this PR is relevant anymore.

**Pull request closed without merging**